docs/appendices/release-notes/4.4.2.rst

```diff
@@ -0,0 +1,59 @@
+.. _version_4.4.2:
+
+=============
+Version 4.4.2
+=============
+
+Released on 2021-03-02.
+
+.. NOTE::
+
+    If upgrading a cluster, you must be running CrateDB 4.0.2 or higher before
+    you upgrade to 4.4.2.
+
+    We recommend that you upgrade to the latest 4.3 release before moving to
+    4.4.2.
+
+    A rolling upgrade from 4.3.0+ to 4.4.2 is supported.
+
+    Before upgrading, you should `back up your data`_.
+
+.. _back up your data: https://crate.io/docs/crate/reference/en/latest/admin/snapshots.html
+
+
```

Comment:
nit: I would cut these two lines. we generally do not add any additional space above the rubric. we do add an extra space before headings